unit-tests/src/test/scala/scala/scalanative/re2s/ExecTest.scala.disabled

```diff
@@ -0,0 +1,633 @@
+package scala.scalanative
```

Comment:
Is this test still failing? Can we enable it?

Reply:
ExecTest.scala itself almost certainly wants to stay disabled. At least until we get some
sort of tiered once-a-week not every CI scheme worked out.  Agreed, a back burner
project.  Those tests are, by design, exhaustive and take tens of minutes (20? 30?)
to run. They are useful enough to be visible to future devos in the Repository, rather
than squirreled away in my boneyard.  Your thoughts?

I believe there are other passing tests which I disabled because Travis was failing when
I enabled the full set.   After I fsync my current batch of changes I an go back and
see if I figure out why Travis is complaining.   Any given N - 2 of the tests pass on
Travis, but running the full N fails.

Reply:
@LeeTibbert Ok, lets keep it disabled for now.